ofrak_core/ofrak/resource.py

```diff
@@ -1464,14 +1466,16 @@ async def delete(self):
         self._resource.is_modified = True
         self._resource.is_deleted = True
 
-    async def flush_to_disk(self, path: str):
+    async def flush_to_disk(self, path: str, pack: Optional[bool] = True):
```

Comment:
We use `Optional` if we need to support `None`. I defer to @EdwardLarson here, but I don't think `None` makes sense here, really. Same goes for the other function signatures.

Since you are giving the parameter a default value of `True`, I don't think we don't need `Optional`.

Reply:
Good catch, I forgot to remove it after I set the default. Fixed it also in `write_to()`